utils/post.js

```diff
@@ -368,7 +370,7 @@ async function filterFeeds(
 
       newItem.is_self = item?.actor?.id === userId || item?.actor?.id === selfAnonymousUserId;
       newItem = await modifyFeedIsFollowingTarget(newItem, userId);
-      newItem = modifyNewItemAnonymity(newItem);
+      newItem = modifyNewItemAnonymity(newItem, isBlurredPost);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyNewItemAnonymity` function call now includes the `isBlurredPost` parameter. This change looks good as long as the `isBlurredPost` value is correctly passed from the calling function. Ensure that this is the case to avoid potential bugs.
```diff
- newItem = modifyNewItemAnonymity(newItem);
+ newItem = modifyNewItemAnonymity(newItem, isBlurredPost);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->